src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +286,40 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was deprecated.
+* File handles can now be serialized and stored in IndexedDB, or sent via `postMessage`
+  to other windows or workers within the same origin. Note that permissions
+  are not retained between browser sessions. For example, when a browser tab
+  is re-opened, and a file handle is obtained from IndexedDB, the user will
+  need to grant permission to read/write to the file again.
+* Added support for `isSameEntry()`, which returns true if two entries
+  represent the same file or directory.
+* Updated usage indicators to indicate whether the user has granted the domain
+  permission to files, including a new read-only indicator.
+
```

Comment:
we also added `resolve()` to figure out if a file/directory is a descendant of a directory, and if so return the relative path from the directory to the entry. (envisioned use case: editor with a "project" opened (a directory) shown in a side bar. Now user opens a file in a file picker, after which the editor might want to highlight the opened file in the project side bar).

Reply:
Added an item for `resolve()`